### PR TITLE
fix for issue 30

### PR DESCRIPTION
**Pull request by dav-sap**
Not tested yet, no live games to test on.
Did not break the archive games

### CODE REVIEW

File: src/games.py [hunk 1]
```diff
@@ -248,13 +263,18 @@ def addGamesLinks(date = '', video_type = "archive"):
...
+                    #     et = calendar.timegm(time.strptime(game['et'], '%Y-%m-%dT%H:%M:%S.%f')) * 1000
                     if add_link == True:
                         params = {
                             'video_id': game_id,
                             'video_type': video_type,
                             'seo_name': seo_name,
                             'has_away_feed': 1 if has_away_feed else 0,
                             'has_condensed_game': 1 if has_condensed_video else 0,
+                            'st' : calendar.timegm(time.strptime(game['st'], '%Y-%m-%dT%H:%M:%S.%f')) * 1000,
+                            'et' : calendar.timegm(time.strptime(game['et'], '%Y-%m-%dT%H:%M:%S.%f')) * 1000,
```
Comment (a7ypically) [inline src/games.py]: You should probably be careful here and first test that game['et'] exists. For live games it probably isn't. I'd also check 'st' just to be on the safe side. I'd set them to None if they don't exist and in getGameUrl() add another check before I use them.

**Comment by dav-sap** [comment]
Tested and not working.. will work on fixes

**Comment by dav-sap** [comment]
Tested & works!
You can start any game from the beginning.
While the game is still on you will be able to start it from the beginning, however the duration of the game will be from the beginning till the present time- meaning it probably won't continue live, so I'm still working on solutions for that. ( I have been thinking what will happen if we just use duration of like 4 hours, will it continue live? probably not need to be checked, any ideas will be welcomed)
Any game in the archive will now start from the beginning no matter what.
Any game in the LIVE section, you can choose to start from the beginning, so games that ended half an hour ago, and are still in the live section you'll be able to watch no problems.

**Comment by maxgalbu** [comment]
Thanks! I'll try to get this change approved in time for the finals

**Comment by Miss80** [comment]
Just a quick question, will it also be possible to start the archive  game from a certain quarter? I know that my boyfriend (sometimes) chooses Q3 to start if he watched part of the game already live during the night (We are from NL so different timezone ;-) )

**Comment by dav-sap** [comment]
@Miss80 Now that I know to work with the "start time" of the link, IF the nba's API has those times it should not be a problem, but that's a big IF that needs to be checked. Start a new issue for this, and I'll try to check it in the next couple of days

**Comment by besarbb** [comment]
How can we add this on our devices, or will we have to wait until an update is available?

**Comment by dav-sap** [comment]
@besarbb you can take it from the branch that did the pull request, but I would wait for it to be merged. After it has been thoroughly tested and reviewed.

**Comment by dav-sap** [comment]
Game doesn't start from the beginning in the middle of the game, just right after it finishes.

**Comment by maxgalbu** [comment]
I'm not able to test it right now, i have merged into master so it can be downloaded from here: https://github.com/maxgalbu/xbmc.plugin.video.nba/releases/download/nightly-master/plugin.video.nba-master.zip

**Comment by dav-sap** [comment]
Like I wrote this morning, from my tests for game 2, the "Start from beginning" in the middle of the game doesn't work, but it has effect after the game is over but still in the live section.
We can git rid of the dialog asking you for options, it makes no difference right now. you want me to commit that?

**Comment by maxgalbu** [comment]
I already merged it. Do you just detect when a game is over using the end date?

**Comment by dav-sap** [comment]
I can't explain why it works. It might be depending of the end_time of a game, it it exists, i use the existing one and maybe that's when the change occurs. Tough to test when there's just one game, and you trying to enjoy the finals as well :)
The way it works now, the "Start from beginning" only works right after the game has ended ( nothing that i do, like I explained), you can leave it like this, it doesn't change much anyway, only problem is, it can be confusing someone might think, he can start from the beginning, but then it will be live.

**Comment by dav-sap** [comment]
I think we'll leave it like this, and next season we'll try to get it to work. At least we can start the game from the beginning right after it finishes.

**Pull request merged**